editor/concrete-grammar/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,29 @@
+# Welcome to your VS Code Extension
```

Comment:
Can we improve this README to be less generic?
It should point to the correct directories and I think a link to our parser can be useful for contributors